### DISCUSSION TITLE
java.lang.ClassNotFoundException: org.apache.flink.shaded.guava18.com.google.common.collect.Lists

### DISCUSSION THREAD

**Discussion by jwsmai** [apache/flink-cdc#2277] [General]
flinkVersion = '1.13.6'
implementation 'com.ververica:flink-sql-connector-mongodb-cdc:2.4.0'

代码编译没问题，启动程序报错：
`Caused by: java.lang.ClassNotFoundException: org.apache.flink.shaded.guava18.com.google.common.collect.Lists`

加上依赖`implementation 'org.apache.flink:flink-shaded-guava:18.0-13.0'`后仍然报错.

**Comment by lucaAce1024** [comment]
同样的问题，求解
